sdk/schemaregistry/schema-registry-avro/src/schemaRegistryAvroSerializer.ts

```diff
@@ -124,14 +124,44 @@ export class SchemaRegistryAvroSerializer {
     return buffer;
   }
 
-  // REVIEW: signature. See serialize and s/serialize into/deserialize from/.
   /**
    * Deserializes a value from a buffer.
    *
-   * @param buffer - The buffer with the serialized value.
+   * @param input - The buffer with the serialized value.
    * @returns The deserialized value.
    */
-  async deserialize(input: Buffer | Blob | Uint8Array): Promise<unknown> {
+  async deserialize(input: Buffer): Promise<unknown>;
```

Comment:
I don't feel like making this a bunch of overloads helps anything, since they all have the same number of parameters and the ref docs aren't meaningfully different (e.g. there aren't detailed separate code samples)

Reply:
Fair enough, done.